drizzle/0007_shocking_rachel_grey.sql

```diff
@@ -0,0 +1,53 @@
+CREATE TABLE IF NOT EXISTS "EmailVerificationToken" (
+	"id" serial PRIMARY KEY NOT NULL,
+	"token" text NOT NULL,
+	"createdAt" timestamp DEFAULT now() NOT NULL,
+	"expiresAt" timestamp NOT NULL,
+	"email" text NOT NULL,
+	"userId" text NOT NULL,
+	CONSTRAINT "EmailVerificationToken_token_unique" UNIQUE("token")
+);
+--> statement-breakpoint
```

Comment:
This migration seems to drop a lot of indexes that we might need. Also the file name is different than the ones we usually have which are descriptive.

Reply:
Yeah saw that. But it's also creating those later.